<ui version="4.0" >
 <class>GTalkSettingsBase</class>
 <widget class="QWidget" name="GTalkSettingsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Google Talk</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="GTalkSettingsTab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="registration" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>330</width>
        <height>659</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Registration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <layout class="QFormLayout" name="formLayout" >
         <property name="fieldGrowthPolicy" >
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="rowWrapPolicy" >
          <enum>QFormLayout::DontWrapRows</enum>
         </property>
         <property name="horizontalSpacing" >
          <number>-1</number>
         </property>
         <property name="verticalSpacing" >
          <number>-1</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="serverLabel" >
           <property name="text" >
            <string>Server:</string>
           </property>
           <property name="buddy" >
            <cstring>serverEntry</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="serverEntry" >
           <property name="text" >
            <string>talk.google.com</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="portEntry" >
           <property name="text" >
            <string>5223</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="usernameLabel" >
           <property name="text" >
            <string>Username:</string>
           </property>
           <property name="buddy" >
            <cstring>usernameEntry</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="usernameEntry" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="passwordLabel" >
           <property name="text" >
            <string>Password:</string>
           </property>
           <property name="buddy" >
            <cstring>passwordEntry</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="passwordEntry" >
           <property name="echoMode" >
            <enum>QLineEdit::PasswordEchoOnEdit</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="portLabel" >
           <property name="text" >
            <string>Port:</string>
           </property>
           <property name="buddy" >
            <cstring>portEntry</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="autoRegister" >
         <property name="text" >
          <string>Auto Register</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>494</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>330</width>
        <height>659</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QCheckBox" name="useEncryption" >
           <property name="text" >
            <string>Use Encryption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ignoreSslErrors" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Ignore SSL Errors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="oldSsl" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Use Old Style SSL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>337</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>useEncryption</sender>
   <signal>clicked(bool)</signal>
   <receiver>ignoreSslErrors</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>57</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>61</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useEncryption</sender>
   <signal>clicked(bool)</signal>
   <receiver>oldSsl</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>93</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel" >
     <x>125</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useEncryption</sender>
   <signal>clicked(bool)</signal>
   <receiver>ignoreSslErrors</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>84</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>94</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useEncryption</sender>
   <signal>clicked(bool)</signal>
   <receiver>oldSsl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>185</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>227</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
